integrations/vip-integrations-config.php

```diff
@@ -0,0 +1,172 @@
+<?php
+/**
+ * Integrations Configuration.
+ *
+ * @package Automattic\VIP\Integrations
+ */
+
+namespace Automattic\VIP\Integrations;
+
+/**
+ * Class for managing configuration of integrations provided by VIP.
+ *
+ * @private
+ */
+class VipIntegrationsConfig {
+	/**
+	 * Directory path where configuration files are stored.
+	 *
+	 * @var string
+	 */
+	private $config_file_dir = ABSPATH . 'config/integrations-config';
+
+	/**
+	 * Structure of the integration config.
+	 *
+	 * @typedef VipIntegrationConfig
+	 *
+	 * @type array (
+	 *   'label'         => string,
+	 *   'org'           => array<string, string>,
+	 *   'env'           => array<string, mixed>,
+	 *   'network_sites' => array<number, array<string, mixed>>,
+	 * )
+	 */
+
+	/**
+	 * Configurations provided by VIP.
+	 *
+	 * @var array <string, array ( VipIntegrationConfig )>
+	 *
+	 * @example
+	 * array(
+	 *   'block-data-api' => array(
+	 *     array(
+	 *       'env'        => array(
+	 *         'status' => 'enabled',
+	 *         'config'  => array(),
+	 *       ),
+	 *     ),
+	 *   ),
+	 *   'parsely' => array(
+	 *     array(
+	 *       'org'        => array( 'status' => 'blocked' ),
+	 *       'env'        => array(
+	 *         'status' => 'enabled',
+	 *         'config'  => array(),
+	 *       ),
+	 *       'network_sites' => array (
+	 *         1 => array (
+	 *           'status' => 'disabled',
+	 *           'config'  => array(),
+	 *         ),
+	 *         2 => array (
+	 *           'status' => 'enabled',
+	 *           'config'  => array(),
+	 *         ),
```

Comment:
I know it's just an example, but I think this is surfacing in the code as a result so wanted to point out here too.

Nothing blocked at the org level should ever make it to the WP site. There's no scenario where we will block on org but enable on env. If blocked at org level or site level, then that should mean the configmap will never see an entry for that integration. It doesn't exist at all.

Reply:
Thanks for pointing this out. Fixed the example in https://github.com/Automattic/vip-go-mu-plugins/pull/5643/commits/fe14e065a2a00eabedf89d70a61b4582cb40a87d

As per current implementation if the integration is blocked on `org` level then we don't include any data from site level integration even if there is data.

```php
// Example

<?php

return array(
  'parsely' => array(
    array(
      'org'  => array( 'status' => 'blocked' ),
    ),
  ),
);
```